src/api/_lib/handlers/push.ts

```diff
@@ -11,23 +12,34 @@ export function getPushRewriteTarget(event: PushEvent): DiscordWebhooksTarget {
 	// Workaround for pre-monorepo
 	if (event.ref === 'refs/heads/v13') return 'discord.js';
 
-	const packages = new Set<PackageName>();
+	const targets = new Set<AppName | PackageName>();
 
 	for (const commit of event.commits) {
+		for (const name of Object.values(AppName)) {
```

Comment:
I noticed we do `Object.values(AppName)` and `Object.values(PackageName)` very frequently, can't we cache them under `AppNameValues` and `PackageNameValues`?